distribution/ddf-common/src/main/resources/bin/ddf

```diff
@@ -51,6 +51,14 @@ managing_solr() {
     return $RC
 }
 
+shutdown_no_java_home() {
+    # Shutdown if JAVA_HOME is not set
+    if [ -z $JAVA_HOME ]; then
+        echo "JAVA_HOME not set. Set JAVA_HOME to proceed - exiting."
+        exit $KARAF_EXEC_RC
```

Comment:
❓ Are we guaranteed that `$KARAF_EXEC_RC` won't be 0 here? Should we have a different error code for this particular error?

Reply:
Good call, fixed